### PR TITLE
Vc02 3537 api spec update

### PR DESCRIPTION
**Pull request by EwaGrela**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.4.2
+- Removing "/dates" endpoint as redundant
+- Updating cli.py to run availability tests depending on type of product
```
Comment (trojkat) [inline CHANGELOG.md:5]: This one belongs to the testing tool.

File: supplier_api_tester/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.4.2
+- Updating availability tests
```
Comment (trojkat) [inline supplier_api_tester/CHANGELOG.md:4]: Removing tests for the list of available dates.

File: CHANGELOG.md (hunk 1 above)
Comment (EwaGrela) [reply CHANGELOG.md:5]: will move it

File: supplier_api_tester/CHANGELOG.md (hunk 2 above)
Comment (EwaGrela) [reply supplier_api_tester/CHANGELOG.md:4]: thanks

File: supplier_api_tester/supplier_api_tester/utils/reservation.py [hunk 3]
```diff
@@ -47,7 +47,12 @@ def get_reservation_slot(api_url, api_key, product_id, timeslots: bool, version=
     return single_variant_item
 
 
-def get_payload_from_slot(slot: Union[DailyAvailability, Timeslot], variant_quantity=1, min_quantity=1):
+def get_payload_from_slot(slot: Union[DailyVariants, Timeslot], variant_quantity=1, min_quantity=1):
+    print()
+    print("payload for slot is called")
+    print()
+    print("slot", slot, type(slot))
+    print()
```
[the remark this answers is not in the document]
Comment (EwaGrela) [reply supplier_api_tester/supplier_api_tester/utils/reservation.py:55]: will remove and also squash it, so the trace of it is gone

[the remark this answers is not in the document]
Comment (EwaGrela) [reply supplier_api_tester/supplier_api_tester/utils/reservation.py:55]: I will remove it and squash the commits, will not be seen again

File: supplier_api_tester/supplier_api_tester/tester.py [hunk 4]
```diff
@@ -30,19 +29,13 @@ def __init__(
 
     def run(self) -> List[TestResult]:
         TEST_TARGETS = {
-            'availability': _run_tests(AVAILABILITY_TEST, self.host, self.api_key, self.product_id),
```
Comment (trojkat) [inline supplier_api_tester/supplier_api_tester/tester.py:33]: We still need to have a way to run only the availability tests.

**Review by trojkat — APPROVED** [review]
:+1:

**Pull request merged**